webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const entryPointNames = [
+	'url',
+	'a11y',
+	'dom-ready',
+];
+
+const externals = {
```

Comment:
How are we expecting this set to be maintained? None of the included packages in this repository use these globals.